### DISCUSSION TITLE
bootstrap5

### DISCUSSION THREAD

**Discussion by olariuromeo** [django-cms/djangocms-template#20] [Q&A]
I don't understand why you don't use bootstrap 5 in this template latest version 4.1? It does not work or what is the reason?
- https://github.com/django-cms/djangocms-template/tree/cms4-1 
- https://github.com/django-cms/djangocms-frontend
- https://github.com/django-cms/djangocms-bootstrap5

**Comment by sgordeychuk** [comment]
@olariuromeo that's why the commit in cms4-1 support has "WIP" in commit message :) it's still in progress and a lot of stuff needs to be changed in the template in general there, switching to bootstrap5 and adding support of djangocms-frontend support is one of the things from ToDo list. The majority of libraries from the template doesn't have django CMS 4.1 support, so they'll need to be either improved or replaced.
